package.js

```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'forwarder:autoform-wizard',
   summary: 'A multi step form component for AutoForm.',
-  version: '0.8.0',
+  version: '0.8.2',
```

Comment:
Let's make this 0.9.0 since this will incompatible with older autoform versions.